provider/cluster_rosa_classic_state.go

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
Suggesting to have this new file under "modules" package and not "provider"
This is an internal module and not provider
Or it can be under "provider/modules"

Reply:
I changed it to be in a different directory